src/main/java/pl/poznan/put/cs/scenariochecker/model/Scenario.java

```diff
@@ -7,7 +7,37 @@
 @Data
 public class Scenario {
     private String title;
-    private List<String> actors;
     private String systemActor;
     private List<Step> steps;
+    private List<String> actors;
+
+    private int numberOfSteps;
+
+    public void setNumberOfSteps(int numberOfSteps) {
```

Comment:
again, constructor, setters and getters are unnecessary. `@Data` annotation does it for us behind the scenes. :)